### ISSUE TITLE
Error when adding File Input field. Thanks

### ISSUE THREAD

**Issue by EllisonTravel** [TiagoSilvaPereira/vemto-issues#357]
```
TypeError: Cannot read property 'nullable' of null
    at Proxy.fixNullableValidationFromField (app://./js/chunk-common.fe76fb8b.js:1:72663)
    at Proxy.fixValidationFromType (app://./js/chunk-common.fe76fb8b.js:1:72562)
    at o.create (app://./js/project_crud.a0c809af.js:1:55315)
    at app://./js/project_crud.a0c809af.js:1:54970
```

---

vemto 1.1.8
Electron 8.5.5
win32 10.0.22000
Locale: en-US

**Comment by TiagoSilvaPereira** [comment]
@EllisonTravel thank you for reporting, we'll fix it ASAP

**Comment by TiagoSilvaPereira** [comment]
Fixed on Vemto 1.1.9
